app/classes/textile.rb

```diff
@@ -114,6 +115,14 @@ def textilize(do_object_links = false, sanitize = true)
 
     # Let Textile munge the thing up now.
     red = RedCloth.new(self)
+
+    # Quote bracketed years
+    # in order to stop RedCloth from turning them into footnote calls
+    # Some Name Citations contain bracketed years. Ex:
+    #   Hyménomycètes (Alençon): 103 (1874) [1878]
+    # We want them to render as such, not footnote references.
+    red.gsub!(BRACKETED_YEAR, '==[==\1]') if red.match?(BRACKETED_YEAR)
+
```

Comment:
I could have sworn this had to come before the RedCloth.new(self) line...  Yet, you have tested it and it apparently works.  I am so confused!

Reply:
Why didn't I think of that? It will be a little simpler that way.
I'll make the change.